dpnp/backend/extensions/statistics/sliding_window1d.hpp

```diff
@@ -574,9 +574,18 @@ void submit_sliding_window1d(const PaddedSpan<const T, SizeT> &a,
             }
 
             auto *const out_ptr = out.begin();
-            auto *const out_end = out.end();
-            results.store(&out_ptr[glid],
-                          [out_end](auto &&ptr) { return ptr < out_end; });
+            // auto *const out_end = out.end();
+
+            auto y_start = glid;
+            auto y_stop =
+                std::min(y_start + WorkPI * results.size_x(), out.size());
+            int32_t i = 0;
+            for (uint32_t y = y_start; y < y_stop; y += results.size_x()) {
+                out_ptr[y] = results[i++];
+            }
+            // due to excessive optimizations this code results in memory
```

Comment:
It would be great to leave any issue number as easy way to find in code when needs to be reverted once resolved.

Reply:
Added mention of SAT-7693